### PR TITLE
Add PHP5.3 support

### PR DESCRIPTION
**Pull request by jairlopez**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,10 +10,10 @@
     }
   ],
   "require": {
-    "php": ">5.6"
+    "php": ">=5.3"
   },
   "require-dev": {
-    "phpunit/phpunit": ">5.0"
+    "phpunit/phpunit": ">=4.8",
```
Comment (jeremyfrench) [inline composer.json]: Trailing comma

**Review by jeremyfrench — CHANGES_REQUESTED** [review]
There is a trailing comma in the composer.json that may be why the tests are not running. Other than that it looks sensible

**Review by jeremyfrench — APPROVED** [review]

**Comment by jeremyfrench** [comment]
Thanks very much for this,

**Comment by stewsnooze** [comment]
nice one @jairlopez

**Comment by jairlopez** [comment]
My pleasure!

Stay safe and healthy.

**Pull request merged**